internal/printer/persistentvolumeclaim.go

```diff
@@ -260,7 +260,8 @@ func getBoundPersistentVolume(ctx context.Context, pvc *corev1.PersistentVolumeC
 
 	pv, err := objectStore.Get(ctx, key)
 	if err != nil {
-		return nil, errors.Wrapf(err, "get volume for key %+v", key)
+		// Return empty persistent volume instead of error in case the dynamic cache has not found the volume yet
+		return persistentVolume, nil
```

Comment:
I remember doing this way long ago to make the educates usage of Octant work, that we have bring this back in to make it work is a sign that things still aren't quite right.

We should be handling these errors in a better way.

Imagine using Octant thinking you had access to PVs or knowing that you created some but you just see empty .. no error, no indication that there was an error of anykind.

Reply:
This behavior is now changed to show the volume as text

![image](https://user-images.githubusercontent.com/10288252/121068928-b36e7000-c781-11eb-8eaf-e902ed0e4c6a.png)